README.md

```diff
@@ -6,7 +6,7 @@ For [general questions](https://community.metamask.io/c/learn/26), [feature requ
 
 MetaMask supports Firefox, Google Chrome, and Chromium-based browsers. We recommend using the latest available browser version.
 
-For up to the minute news, follow our [Twitter](https://twitter.com/metamask) or [Medium](https://medium.com/metamask) pages.
+For up to the minute news, follow us on [X](https://x.com/MetaMask).
```

Comment:
@davidmurdoch Did you want to remove `or [Medium](https://medium.com/metamask) pages.` this ?

Reply:
Yes. It hasn't been updated in years.